### PR TITLE
Align operationIds to comply with guidelines

### PR DESCRIPTION
**Pull request by MisterKite**
Changed operationIds to reflect Guidelines + updated guideline

### CODE REVIEW

File: maturity-level-one/README.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ Documentation should provide information about the following at least:
 ### Defining OperationIds
 An operationId is a unique identifier for an operation that is provided on an API. It is important to think carefully when assigning an operationId as changing these later on will be a breaking change.
 
-OpenAPI tooling, such as [AutoRest](https://github.com/Azure/autorest) & [NSwag](https://github.com/RSuter/NSwag), use the operationId to generate API clients based on this convention so it is important to provide a descriptive operationId. In order to ease the use of your API we recommend using the `{controller}_{operationName}` pattern.
+OpenAPI tooling, such as [AutoRest](https://github.com/Azure/autorest) & [NSwag](https://github.com/RSuter/NSwag), use the operationId to generate API clients based on this convention so it is important to provide a descriptive operationId. In order to ease the use of your API we recommend using the `{controller}_{operationName}` pattern in Pascal Case.
```
Comment (tomkerkhove) [inline maturity-level-one/README.md]: Maybe improve a bit for readability such as `in pascal casing`?

File: maturity-level-one/src/Controllers/PlayersController.cs [hunk 2]
```diff
@@ -81,7 +77,7 @@ public async Task<IActionResult> Create(Player player)
         }
 
         [HttpPut("{id}")]
-        [SwaggerOperation("update-player")]
+        [SwaggerOperation("Players_Update")]
```
Comment (tomkerkhove) [inline maturity-level-one/src/Controllers/PlayersController.cs]: Should be UpdateFull to align with operation name?

File: maturity-level-one/src/Controllers/PlayersController.cs [hunk 3]
```diff
@@ -100,7 +96,7 @@ public async Task<IActionResult> UpdateFull(int id, [FromBody] PlayerDto player)
         }
 
         [HttpPatch("{id}")]
-        [SwaggerOperation("update-player-incremental")]
+        [SwaggerOperation("Players_UpdateIncremental")]
         [SwaggerResponse(204, "No Content")]
         [SwaggerResponse(404, "Player not found")]
         [SwaggerResponse(500, "API is not available")]
```
Comment (tomkerkhove) [inline maturity-level-one/src/Controllers/PlayersController.cs]: @MassimoC 	Did you implement this? If so, shouldn't this be a patch and UpdateFull be renamed to Update?

File: maturity-level-one/src/Controllers/PlayersController.cs (hunk 2 above)
Comment (MisterKite) [reply maturity-level-one/src/Controllers/PlayersController.cs]: Yes, oversight on my part. Will change

File: maturity-level-one/src/Controllers/PlayersController.cs (hunk 3 above)
Comment (tomkerkhove) [reply maturity-level-one/src/Controllers/PlayersController.cs]: Taking this offline with Massimo.

File: maturity-level-one/src/Controllers/PlayersController.cs [hunk 4]
```diff
@@ -112,15 +108,15 @@ public async Task<IActionResult> UpdateIncremental(int id, [FromBody] PlayerDto
...
+            var playerToBeUpdated = Mapper.Map<Player>(player);
             playerToBeUpdated.Id = id;
 
             await _worldCupRepository.ApplyPatch<Player, PlayerDto>(playerToBeUpdated, player);
             return NoContent();
         }
 
         [HttpPatch("{id}/update")]
-        [SwaggerOperation("update-player-jsonpatch")]
+        [SwaggerOperation("Players_Patch")]
```
Comment (MassimoC) [inline maturity-level-one/src/Controllers/PlayersController.cs]:
```suggestion
        [SwaggerOperation("Players_UpdateIncrementalJsonPatch")]
```

Comment (tomkerkhove) [reply maturity-level-one/src/Controllers/PlayersController.cs]: That's good for me as well, as long as it reflects in the method name.

**Comment by tomkerkhove** [comment]
@MassimoC 	@MisterKite 	what's the status here?

**Comment by tomkerkhove** [comment]
Can you also provide an example of the Swagger spec that is generated please @MisterKite?

**Pull request merged**